support/src/setupProxy.js

```diff
@@ -55,10 +113,10 @@ const getDrugSearch = async (req, res, next) => {
 		'drugdictionary',
 		'v1',
 		'Drugs',
-		'Search',
+		'search',
```

Comment:
Why is search here?

Reply:
That got in there from setting up the app in the first place to make sure the fetch calls and mocks work. Just got changed since the casing on the api endpoints changed.